packages/report-server/src/reportBuilder/customReports/testCustomReport.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2022 Beyond Essential Systems Pty Ltd
+ */
+
+import { FetchReportQuery } from '../../types';
+import { ReqContext } from '../context';
+
+export const testCustomReport = async (reqContext: ReqContext, query: FetchReportQuery) => {
+  const { organisationUnitCodes: entityCodes, hierarchy } = query;
+  const facilities = await reqContext.services.entity.getDescendantsOfEntities(
+    hierarchy,
+    entityCodes,
+    { filter: { type: 'facility' } },
+  );
+  return [{ value: facilities.length }];
+};
```

Comment:
Added this `testCustomReport` as an example for future use and a bit of a proof of concept. @chris-bes feel free to remove this once you add your actual implementation 👍

Reply:
@chris-pollard

Reply:
Thanks @rohan-bes